CopilotKit/packages/react-core/src/lib/copilot-task.ts

```diff
@@ -19,26 +19,39 @@ export interface CopilotTaskConfig {
   includeCopilotReadable?: boolean;
 
   /**
-   * Whether to include actions defined via useMakeCopilotActionable in the task.
+   * Whether to include actions defined via useCopilotAction in the task.
+   * @deprecated Use the `includeCopilotActions` property instead.
    */
   includeCopilotActionable?: boolean;
+
+  /**
+   * Whether to include actions defined via useCopilotAction in the task.
+   */
+  includeCopilotActions?: boolean;
 }
 
 export class CopilotTask<T = any> {
   private instructions: string;
   private actions: FrontendAction<any>[];
   private includeCopilotReadable: boolean;
-  private includeCopilotActionable: boolean;
+  private includeCopilotActions: boolean;
 
   constructor(config: CopilotTaskConfig) {
     this.instructions = config.instructions;
     this.actions = config.actions || [];
-    this.includeCopilotReadable = config.includeCopilotReadable || true;
-    this.includeCopilotActionable = config.includeCopilotActionable || true;
+    this.includeCopilotReadable =
+      config.includeCopilotReadable !== undefined ? config.includeCopilotReadable : true;
+    if (config.includeCopilotActions !== undefined) {
```

Comment:
a little complex, perhaps we can simplify?
with the right order, no need for else cases etc.

```
// true by default, conditionally override the value if configs are set:
this.includeCopilotActions = true
if(config.includeCopilotActionable !== undefined) {  //
   this.includeCopilotActions = config.includeCopilotActionable
}
if(config.includeCopilotActions !== undefined) { 
    this.includeCopilotActions = config.includeCopilotActions
}
```

Reply:
```
    this.includeCopilotReadable = config.includeCopilotReadable !== false;
    this.includeCopilotActions =
      config.includeCopilotActions !== false && config.includeCopilotActionable !== false;
 ```